using-babelfish/clients.md

```diff
@@ -49,10 +51,38 @@ func main() {
 		panic(err)
 	}
 
-	query := "//*[@role='Identifier' and not(@role='Qualified')]"
+	query := "//*[not(@token = '') and not(@role='Qualified')]"
 	it, _ := tools.Filter(res, query)
 	for it.Next() {
-		fmt.Println(it.Node())
+		node, ok := it.Node().(nodes.Object)
```

Comment:
Oh, sorry, haven't noticed - this check is a bit to early and will not allow any arrays.

Reply:
Fixed, thanks!